### ISSUE TITLE
Collaborate with story telling team to share learnings from our A11y statement process

### ISSUE THREAD

**Issue by YedidaZalik** [cds-snc/notification-planning#1583]
As CDS or public service staff, I need to be able to create accessible products so that these products can be used by everyone for whom they are intended.

WHY are we building?
We learned so much from updating our A11y statement that could be helpful to other products.

WHAT are we building?
A blog post that describes how updating our Accessibility statement led to a more accessible Notify

VALUE created by our solution
Other teams can use our learnings 

## Documentation and Artifacts
[Blog on GC Notify's accessibility statement](https://docs.google.com/document/d/1OhUByoPQyikT-axG6mBQzilBZNGBGksuz4bCSH5t_Bc/edit)

## Acceptance Criteria

- [x] Story telling and content design collaborate on idea for blog
- [x] Story telling interviews applicable disciplines on team (Yael, Melissa, Phil, Andrew, Marie-Sophie and Yedida)
- [x] Content designer reviews and assists with revisions
- [x] PM reviews and any changes she indicates are incorporated
- [x] Copy is sent to Josh for FYI
- [x] Localization is completed
- [ ] Publication

* A11y
* Bilingualism

* Measuring success and metrics

**Comment by yaelberger-commits** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/1583?planning-poker) with Zenhub @YedidaZalik

**Comment by YedidaZalik** [comment]
Met with Jo to refine story and added links to Jo's template last weekhttps://docs.google.com/document/d/1OhUByoPQyikT-axG6mBQzilBZNGBGksuz4bCSH5t_Bc/edit

**Comment by YedidaZalik** [comment]
Yedida reviewed Jo's drafts and asked some questions

**Comment by amazingphilippe** [comment]
Moving to blocked until we hear back from storytelling team

**Comment by YedidaZalik** [comment]
New outline from Jo yesterday
Yedida to review and make suggestions

**Comment by YedidaZalik** [comment]
Reviewed and make suggestions

**Comment by yaelberger-commits** [comment]
I'm reviewing the draft outline today

**Comment by amazingphilippe** [comment]
Lots of suggestions accepted by Jo, writing more examples today before Yael review

**Comment by YedidaZalik** [comment]
Yael reviewed.
Jo & Yedida starting Interviews with team members today.

**Comment by YedidaZalik** [comment]
Completed interview with Melissa yesterday.

**Comment by YedidaZalik** [comment]
Have been editing the blog, spent most of yesterday on it and will spend time today

**Comment by YedidaZalik** [comment]
Blog now with relevant Notify team members for review by Aug 14

**Comment by YedidaZalik** [comment]
Jo accepted edits; now with her for next steps

**Comment by YedidaZalik** [comment]
Slacked Jo to ask her to flag blog to Lesley before translating 
(Lesley since Josh is away)

**Comment by yaelberger-commits** [comment]
Translation completed. 
Phil created some screenshots to include with the post. Aiming to publish today.
